README.md

```diff
@@ -72,11 +72,7 @@ We first show how to apply Dredd to a simple stand-alone program. We will then s
 
 ```
 # This will modify pi.cc in-place.
-# The --mutation-info-file argument is used to specify a JSON file to
-# which Dredd will output machine-readable information about the
-# mutations it applied. For the purposes of this example, this file
-# can be ignored.
-${DREDD_EXECUTABLE} examples/simple/pi.cc --mutation-info-file temp.json
+${DREDD_EXECUTABLE} examples/simple/pi.cc
```

Comment:
Two things:

- Below there is an `rm temp.json` that should go.

- Further down still, in the part about mutating code that uses a CMake project, the `--mutation-info-file` argument is used. Can you migrate this (currently removed) comment explaining the purpose of this argument to lower down, so that the argument is still explained when it is used?